lib/XML/Feed/Format/MRSS.pm

```diff
@@ -0,0 +1,77 @@
+package XML::Feed::Format::MRSS;
+use strict;
+use warnings;
+
+use base qw( XML::Feed::Format::RSS );
+
+use XML::Feed::Entry::Format::MRSS;
+
+use XML::FeedPP;
+
+sub format { 'MRSS' }
+
+sub identify {
+    my $class   = shift;
+    my $xml     = shift;
+
+    my $feed = XML::FeedPP->new( $$xml );
+    my $is_mrss = 1;
+    if ( index($$xml, "xmlns:media=\"http://search.yahoo.com/mrss/\"" ) < 0 ) {
+        $is_mrss = 0;
+    }
+
+    foreach my $item ( $feed->get_item() ) {
+        if ( ! $item->{'media:content'}  )  {
+            $is_mrss = 0;
```

Comment:
changed check to indicate mrss if  there is an $item->{'media:content'} element AND a $item->{'media:content'}->{'-url'} within that element. is that ok?